tests/AbstractTestCaseWithOldNow.php

```diff
@@ -29,10 +29,9 @@ abstract class AbstractTestCaseWithOldNow extends AbstractTestCase
     protected function tearDown(): void
     {
         Carbon::resetMacros();
-        Carbon::setTestNow();
```

Comment:
Both `*:: setTestNow();` can safely be removed as they are already call in `\Tests\AbstractTestCase::tearDown`